Frontend/src/styles/Header.css

```diff
@@ -51,4 +51,37 @@ nav img {
 
 .header-icon {
   width: 54px;
+}
+
+/* User dropdown styling */
+.dropdown {
+  position: relative;
+  display: inline-block;
+  height: 100%;
+}
+
+.dropdown-content {
+  display: none;
+  position: absolute;
+  right: 0;
+  top: 54px;
+  background-color: var(--background);
+  min-width: 160px;
+  box-shadow: 0px 8px 16px -1px var(--light-shadow);
+  z-index: 1;
+}
+
+.dropdown-content a {
+  color: black;
+  padding: 12px 16px;
+  text-decoration: none;
+  display: block;
+}
+
+.dropdown-content a:hover {
```

Comment:
Should really add `cursor: pointer;` to this.

Reply:
Changed it so that cursor changes to pointer on hover over logout. Pushed it to the branch, hopefully it updated.

Reply:
It doesn't show any new commits